.travis.yml

```diff
@@ -1,9 +1,14 @@
 language: java
-sudo: false
+# Use Ubuntu Trusty with sudo instead of new default Trusty because of higher resources availability
+sudo: required
+dist: trusty
```

Comment:
use 

`dist: xenial`

because trusty is deprecated
https://docs.travis-ci.com/user/reference/xenial/